### PR TITLE
Added current schemas to the documentation

### PR DESCRIPTION
**Pull request by joselsegura**

### CODE REVIEW

File: docs/internal-pipeline/parquet_features.md [hunk 1]
```diff
@@ -96,7 +96,11 @@ An example of UUID:
 ## Format of the `report` node
 
 `report` node is represented as a list (or rather an array) of zero or more
-items. Each item has the following attributes:
+items. These items represent the result of the execution of each rule configured
+in the "feature extraction service", so their format will depend on the defined
```
Comment (tisnik) [inline docs/internal-pipeline/parquet_features.md:100]: defined _what_?

**Review by tisnik — APPROVED** [review]
LGTM

File: docs/internal-pipeline/parquet_features.md (hunk 1 above)
Comment (joselsegura) [reply docs/internal-pipeline/parquet_features.md:100]: Missing word, fixed

**Comment by joselsegura** [comment]
TBH I don't know why the `schemas/` validators scripts are shown in the diff, but it should'nt, no changes compared to current master branch ones

**Review by Bee-lee — APPROVED** [review]
lgtm

**Pull request merged**